src/components/Kv/KvButton.vue

```diff
@@ -108,6 +108,7 @@ export default {
 	background: transparent;
 	font-size: 1em;
 	transition: none;
+	vertical-align: baseline;
```

Comment:
@ryan-ludwig I had to add this to get the kvButton text-link to align with text in spans in the settings cards. I don't believe this will have adverse effects anywhere. Previous inherited value was `middle`

Reply:
Cool, this should be fine.